### PR TITLE
add march hackathon event page

### PR DESCRIPTION
**Pull request by mashehu**
More or less copy-pasted from the last hackathon page. unix timestamps need to be updated, but I first want to know which time slots we actually are using.

### CODE REVIEW

File: markdown/events/2022/hackathon-march-2022.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+start_date: "2022-03-16"
+start_time: "10:00 CET"
+end_date: "2022-03-18"
+end_time: "18:00 CET"
+location_name: Gather town and Slack.
+---
+
+
+Please join us for the March 2022 nf-core hackathon!
+This event is entirely virtual (we will be back in gather.town!) and we are attempting to support people in all timezones.
```
Comment (ewels) [inline markdown/events/2022/hackathon-march-2022.md:14]:
```suggestion
This event is entirely virtual (we will be back in [gather.town](https://gather.town/)!) and we are attempting to support people in all timezones.
```

File: markdown/events/2022/hackathon-march-2022.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+More details will come soon, registration will open February 2022.
+Please check back here for details and keep an eye out on twitter and on Slack,
+but in the mean time - book the dates in your calendar!
+
+# Schedule
+
+We expect people to come and go during the hackathon due to diverse time zones.
+Please just do whatever works best for you!
+
+All locations described in the table below refer to places in the Hackathon gather.town
```
Comment (ewels) [inline markdown/events/2022/hackathon-march-2022.md:26]:
```suggestion
All locations described in the table below refer to places in the Hackathon [gather.town](https://gather.town/)
```

File: markdown/events/2022/hackathon-march-2022.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+space.
+
+The following schedule should display times in your local time zone:
+
```
Comment (ewels) [inline markdown/events/2022/hackathon-march-2022.md:30]:
```suggestion
The following schedule should display times in your local time zone:

> **NB: This is a draft schedule and subject to change**

```

File: markdown/events/2022/hackathon-march-2022.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+## Social Activities
+
+During the hackathon, we will have a few light-hearted fun and games!
+
+- Breaks will happen in the dedicated Cafeteria room, for informal chatting and getting to know each other.
+<!-- - Throughout the three days, we will once again be running a nf-core hackathon **bingo**! To join the game, you can go the following [link](https://nfcore-bingo.web.app/?game=nf-core-hackathon). Check the instructions at the bottom of the page.
+
+    > <i class="fas fa-hand-paper"></i> Bingo! <https://nfcore-bingo.web.app/?game=nf-core-hackathon>
+
+- In addition, we will be running an easter-egg (sock? 😉) hunt! There are 11 socks distributed around the gather.town world. Take screenshots of as many as you can find! -->
```
Comment (ewels) [inline markdown/events/2022/hackathon-march-2022.md:253]:
```suggestion
```

File: markdown/events/2022/hackathon-march-2022.md (hunk 3 above)
Comment (ewels) [reply markdown/events/2022/hackathon-march-2022.md:30]: I half wonder if we just comment out the entire schedule until @ggabernet has looked over it. But maybe it's fine. Happy to merge and we can always take it out again.

**Review by ewels — APPROVED** [review]
Nice, LGTM ✅

File: markdown/events/2022/hackathon-march-2022.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+space.
+
+The following schedule should display times in your local time zone:
+
+<div class="table-responsive">
+    <table class="table table-hover table-sm table-bordered">
+        <thead>
+            <tr>
+                <th>Time</th>
+                <th>Weds. 16 Mar., 2022</th>
```
Comment (ggabernet) [inline markdown/events/2022/hackathon-march-2022.md:36]:
```suggestion
                <th>Wed. 16 Mar., 2022</th>
```

File: markdown/events/2022/hackathon-march-2022.md [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+                <td data-timestamp="1635348600" data-timeformat="HH:mm z">17:30</td>
+                <!-- <td rowspan="2">Pair Programming (Americas)<br>
+                   <ul class="small list-unstyled">
+                    <li>Location: Mentor specific rooms</li>
+                  </ul></td> -->
+                <td rowspan="2">Social Event<br>
+                   <ul class="small list-unstyled">
+                    <li>Location: Lounge</li>
+                    <li>Host: Maxime Garcia / Susanna Marquez</ul></td>
+                    <td rowspan="6"></td>
```
Comment (ggabernet) [inline markdown/events/2022/hackathon-march-2022.md:217]:
```suggestion
                <!-- <td rowspan="2">Social Event<br>
                   <ul class="small list-unstyled">
                    <li>Location: Lounge</li>
                    <li>Host: Maxime Garcia / Susanna Marquez</ul></td>
                    <td rowspan="6"></td> -->
```

File: markdown/events/2022/hackathon-march-2022.md [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+                    <!-- <li>Host(s): Harshil Patel, Matthias Hörtenhuber </li> -->
+                  </ul>
+                </td>
+            </tr>
+            <tr>
+                <td data-timestamp="1635348600" data-timeformat="HH:mm z">17:30</td>
+                <!-- <td rowspan="2">Pair Programming (Americas)<br>
+                   <ul class="small list-unstyled">
+                    <li>Location: Mentor specific rooms</li>
+                  </ul></td> -->
```
Comment (ggabernet) [inline markdown/events/2022/hackathon-march-2022.md:212]:
```suggestion
                <td rowspan="2">Group distribution (EMEA)<br>
                   <ul class="small list-unstyled">
                    <li>Location: Lecture Hall</li>
                  </ul></td>
```

**Review by ggabernet — COMMENTED** [review]
Hi @mashehu ,
thanks for the first draft already. I just thought about removing the social event still until we agree on the specific time, but already add a Group distribution session right after the Welcome. The rest looks good to me!

**Pull request merged**